src/Recoil.rei

```diff
@@ -63,11 +63,7 @@ module RecoilRoot: {
 
   [@react.component] [@bs.module "recoil"]
   external make:
-    (
-      ~initialState: option(initializeState)=?,
-      ~children: React.element,
-      unit
-    ) =>
+    (~initialState: initializeState=?, ~children: React.element) =>
```

Comment:
Can the trailing `unit` be omitted given `initialState` is optional? I do that out of habit but I'm actually not sure 🤔

Reply:
(Nevermind I just checked, I guess it can 😄)